server/cache/resource_cache.go

```diff
@@ -17,6 +17,7 @@ type ResourceCache struct {
 	cache  Interface
```

Comment:
remove `ctx` - structs should not store `ctx`

Reply:
done